app/pages/links/links.php

```diff
@@ -0,0 +1,30 @@
+<?php restrict_access();
+$user = User::getCurrent();
+$jot_params = "?nom=" . $user->last_name . "&prenom=" .
+    $user->first_name . "&email=" . $user->real_email;
+$jot_link = new Link(
```

Comment:
This might be something left from testing :) remove?

Reply:
No it's the JotForm Feature ;)